org.mixedrealitytoolkit.input/InteractionModes/InteractionModeManager.cs

```diff
@@ -71,7 +71,7 @@ public static InteractionModeManager Instance
         public void InitializeControllers()
         {
             controllerMapping.Clear();
-            foreach (XRController xrController in FindObjectsOfType<XRController>())
+            foreach (XRController xrController in FindObjectUtility.FindObjectsOfType<XRController>())
```

Comment:
Looks like the Input package.json wasn't updated

Reply:
I think it was already bumped and hopefully this should go in before the next release

Reply:
We haven't shipped and are not shipping Input 3.0.1 yet.   The package.json is already 3.0.1, last shipped version was 3.0.0

Reply:
Oops! Missed that